pkg/health/health_test.go

```diff
@@ -0,0 +1,129 @@
+package health
+
+import (
+	"testing"
+
+	. "github.com/anthonybishopric/gotcha"
+	"github.com/hashicorp/consul/api"
+	"github.com/square/p2/pkg/kp"
+)
+
+func TestFindWorst(t *testing.T) {
+	a := Result{
+		ID:     "testcrit",
+		Status: Critical,
+	}
+	b := Result{
+		ID:     "testwarn",
+		Status: Warning,
+	}
+	c := Result{
+		ID:     "testpass",
+		Status: Passing,
+	}
+
+	id, _ := FindWorst([]Result{a, b})
+	Assert(t).AreEqual(id, a.ID, "FindWorst should have returned testcrit")
```

Comment:
I think we can do with a bit of descriptiveness like "FindWorst between Critical and Warning should have returned testcrit"